### ISSUE TITLE
Fix COLOR CONTRAST a11y issues in org homepage

### ISSUE THREAD

**Issue by christiemolloy** [patternfly/patternfly-org#1351]
https://www.patternfly.org/v4/

There is 1 type of a11y issue:
1. Elements must have sufficient color contrast

Gina and Margot are handling the new designs and then will hand it off to the developer.

**Comment by christiemolloy** [comment]
@rachael-phillips this violation is "serious" but at the same time its a big issue that we need to get started on.

**Comment by christiemolloy** [comment]
After speaking with @gdoyle1 and @mmenestr, and them confirming with @kybaker we have decided to update the background gradient on the home page to be: `linear-gradient(#004080, #0066cc)` . The top section should be darker at the top, and bottom section should be lighter at the top.

**Comment by christiemolloy** [comment]
<img width="1122" alt="Screen Shot 2019-07-18 at 2 36 29 PM" src="https://user-images.githubusercontent.com/20118816/61484444-f98f7000-a96c-11e9-83e7-83fd25133593.png">
<img width="1121" alt="Screen Shot 2019-07-18 at 2 36 41 PM" src="https://user-images.githubusercontent.com/20118816/61484459-fd22f700-a96c-11e9-93a7-db37eef1d3f5.png">
